### PR TITLE
LPD-37004  | Test Fix | Poshi >>> Playwright

### PR DESCRIPTION
**Pull request by rafaellajordao**
### Motivation:
Test failing due to Poshi limitations.

### Proposed Solution:
Transcribe test to playwright in order to facilitate maintenance.

**More details:** [LPD-37004](https://liferay.atlassian.net/browse/LPD-37004)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/utils/utils.ts [hunk 1]
```diff
@@ -58,3 +58,45 @@ export async function searchByTerm({
...
+	paginationResults,
+}: {
+	page: Page;
+	paginationResults: string;
+}) {
+	await waitForLoading(page);
+	await expect(page.locator('.pagination-results')).toContainText(
+		paginationResults
+	);
+	await waitForLoading(page);
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/utils/utils.ts:73]: @rafaellajordao, can you add a breakline before this line?

File: modules/test/playwright/tests/osb-faro-web/utils/utils.ts [hunk 2]
```diff
@@ -58,3 +58,45 @@ export async function searchByTerm({
...
+}
+
+export async function selectPaginationItemsPerPage({
+	itemsPerPage,
+	page,
+}: {
+	itemsPerPage: string;
+	page: Page;
+}) {
+	await waitForLoading(page);
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/utils/utils.ts:83]: @rafaellajordao the same as above

File: modules/test/playwright/tests/osb-faro-web/utils/utils.ts [hunk 3]
```diff
@@ -58,3 +58,45 @@ export async function searchByTerm({
...
+}
+
+export async function selectPaginationPageNumber({
+	page,
+	paginationPageNumber,
+}: {
+	page: Page;
+	paginationPageNumber: string;
+}) {
+	await waitForLoading(page);
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/utils/utils.ts:96]: @rafaellajordao the same as above

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
> ### ❌ ci:test:analytics-cloud - 239 out of 255 jobs passed in 4 hours 5 minutes
> Click here for more details.

the tests modified in this PR are passing correctly! :smile:

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**